modules/manage/partials/audit-logging.adoc

```diff
@@ -397,7 +397,15 @@ ifdef::env-cloud[]
 
 == Configure retention for audit logs
 
-Assess the retention needs for your audit logs. You may not need to keep the logs for the default seven days. This is controlled by setting the `retention.ms` property for the `_redpanda.audit_log` topic.
+Ideally, you export audit events to your own SIEM for persistence supporting your audit and compliance needs rather than relying on in-cluster persistence.
+
+As you assess the retention needs for your audit logs, if you find that you do not need to keep them for the default seven days, and you must revise the retention period:
+
+NOTE: Redpanda Data recommends that you retain audit logs for at least one year.
```

Comment:
Redpanda Data recommends that you retain audit logs for at least one year `in a separate system like your SIEM, so, in case there's an issue with the Redpanda cluster you still have access to the audit logs`